apps/timetable/admin/index.html

```diff
@@ -0,0 +1,150 @@
+            <% } %>
+        </script>
+
+        <script id="gh-help-template" type="text/template">
+            <div id="gh-help">
+                <p>Need help? Contact: <a href="mailto:tthelp@cam.ac.uk" title="tthelp@cam.ac.uk">tthelp@cam.ac.uk</a></p>
+            </div>
+        </script>
+
+        <script id="gh-tripos-help-template" type="text/template">
+            <p class="gh-tripos-help">To start editing timetables choose a tripos from the list above, or select from your list below</p>
+        </script>
+
+        <script id="gh-login-template" type="text/template">
+            <div id="gh-login-form" class="pull-left">
+                <form class="gh-signin-form" role="form">
+                    <h2>Sign in</h2>
+                    <div class="form-group">
+                        <label for="gh-signin-email">Email address</label>
+                        <input type="username" name="username" class="form-control" id="gh-signin-email" placeholder="Enter your email" required>
+                    </div>
+                    <div class="form-group">
+                        <label for="gh-signin-password">Password</label>
+                        <input type="password" name="password" class="form-control" id="gh-signin-password" placeholder="Enter your password" required>
+                    </div>
+                    <button type="submit" id="gh-signin" class="btn btn-default gh-btn-primary">
+                        <i class="fa fa-sign-in"></i>Sign in
+                    </button>
+                </form>
+            </div>
+        </script>
+
+        <script id="gh-modules-template" type="text/template">
+            <%= _.partial('modules', {'data': data}) %>
+        </script>
+
+        <script id="gh-subheader-part-template" type="text/template">
+            <%= _.partial('subheader-part', {'data': data}) %>
+        </script>
+
+        <script id="gh-subheader-picker-template" type="text/template">
+            <%= _.partial('subheader-picker', {'data': data}) %>
+        </script>
+
+        <script id="gh-subheader-pickers-template" type="text/template">
+            <a class="pull-right gh-show-video" href="#" style="display: none;"><i class="fa fa-youtube-play"></i> Help video<a>
+            <%= _.partial('subheader-pickers') %>
+        </script>
+
+        <script id="gh-main-tripos-template" type="text/template">
+            <div id="gh-main-tripos">
+                <div class="gh-video">
+                    <button class="btn gh-hide-video"><i class="fa fa-remove"></i></button>
+                    <span class="gh-video-info">A quick look at how to use<br/>the new timetable system</span>
+                    <button class="btn gh-play-video"><i class="fa fa-youtube-play"></i>Play video</button>
+                </div>
+                <div class="row gh-row-part">
+                    <div class="col-md-4 gh-part">
```

Comment:
I agree that we should make it as flexible as possible using the bootstrap columns. We'll need some structural changes (later, maybe) for this to take full effect though as a min-width is set to the content at the moment.